analyzer/protocol/facefish_rootkit/Readme.md

```diff
@@ -0,0 +1,73 @@
+# Facefish_Rootkit
+
+This Spicy analyzer will detect the Facefish Linux rootkit C2 as described in:
+
+- https://blog.netlab.360.com/ssh_stealer_facefish_en/
+- https://blogs.juniper.net/en-us/threat-research/linux-servers-hijacked-to-implant-ssh-backdoor
+- https://thehackernews.com/2021/05/researchers-warn-of-facefish-backdoor.html
+- https://securityaffairs.co/wordpress/118388/malware/facefish-backdoor.html
+
+Any detections will be found in "facefish_rootkit.log" while a "Facefish_Rootkit::FACEFISH_ROOTKIT_C2" notice 
```

Comment:
```suggestion
Any detections will be found in "facefish_rootkit.log" while a "Facefish_Rootkit::FACEFISH_ROOTKIT_C2" notice
```

Reply:
This is actually a natural space in the sentence.